scripts/build/docker/Dockerfile.template

```diff
@@ -16,6 +16,8 @@ RUN apt-get update \
     curl \
     fuse \
     git \
+    graphicsmagick \
```

Comment:
> mis-identified the ARCH and built a x64 native-module for the x86 Etcher

Solution `gcc-multilib g++-multilib` is tested and proven to work. I am not 100%, but you can check :) (using electron-builder, of course).

> and testing

Our docker image contains Chrome to test ;)